scripts/src/errors.ts

```diff
@@ -116,3 +166,155 @@ export class ChannelBusyError extends Error implements KinSdkError {
 		super('Cannot acquire a free channel.');
 	}
 }
+
+export class BadRequestError extends HorizonError {
+	readonly type: ErrorType = 'BadRequestError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
+		super(`Bad Request error`, errorBody, title);
+	}
+}
+
+export class InternalError extends HorizonError {
+	readonly type: ErrorType = 'InternalError';
+
+	constructor(readonly errorBody: any, readonly title?: string) {
+		super(`internal error`, errorBody, title ? title : "{'internal_error': 'unknown horizon error'}");
+	}
+}
+
+export class AccountExistsError extends HorizonError {
+	readonly type: ErrorType = 'AccountExists';
+
+	constructor(readonly errorBody: any, readonly title?: string) {
+		super(`account already exists`, errorBody, title);
+	}
+}
+
+export class LowBalanceError extends HorizonError {
+	readonly type: ErrorType = 'LowBalanceError';
+
+	constructor(readonly errorBody: ErrorResponse, readonly title?: string) {
```

Comment:
no need for title

Reply:
I can remove it but to keep the code consistent, I prefer to leave it as it is.